db/migrate/20230313163130_create_bookings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateBookings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bookings do |t|
+      t.date :start_date
+      t.date :end_date
+      t.string :status
```

Comment:
maybe set a default value to "pending"? Make sure to migrate again in order to update the schema